davealex/quick-credit
# #165586458 Design Privacy & Terms Page

#### What does this PR do?
This PR adds a privacy_terms.html page to the Project
#### Description of Task to be completed?
On this page, a user can see a simple page with the platform’s privacy & terms related texts
#### How should this be manually tested?
Navigate to the privacy_terms.html page and ensure that the page is properly rendered when served on mobile devices
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#165586458
#### Screenshots (if appropriate)
#### Questions:
